corehq/apps/es/tests/utils.py

```diff
@@ -61,10 +61,18 @@ def es_test(test):
     return attr(es_test=True)(test)
 
 
-def populate_user_index(user_docs):
+def populate_es_index(models, index_name, doc_prep_fn=lambda doc: doc):
+    index_info = ES_META[index_name]
     es = get_es_new()
-    ensure_index_deleted(USER_INDEX)
```

Comment:
is it ok to remove this line?

Reply:
It should be - this is used in two places and both also include this in their teardown.  The risk is there's someplace else that doesn't, but if the test suite passes I assume it's fine.